oasislmf/exposures/csv_trans.py

```diff
@@ -18,6 +18,8 @@
     multithread,
     Task,
 )
+from oasislmf.utils.exceptions import OasisException
```

Comment:
An absolute isn't required here - it's simpler to use a relative import here

    from ..utils.exceptions import OasisException